### DISCUSSION TITLE
Extending the expiration time of confirmation link

### DISCUSSION THREAD

**Discussion by daniel-krastev** [supabase/supabase#16005] [Questions]
Hi,

I am using a confirmation link with the PKCE flow on a server side which seems to expire for 5 minutes, as it is mentioned in the documentation [here](https://supabase.com/docs/guides/auth/server-side-rendering#understanding-the-authentication-flow) under PKCE.

> info
> 
> For security purposes, the code has a validity of 5 minutes and can only be exchanged for an access token once. You will need to restart the authentication flow from scratch if you wish to obtain a new access token.

Is it possible that this could be extended? 5 minutes seems a little aggressive.

Thanks

**Comment by J0** [comment]
Hey thanks!

We're working on extending this: https://github.com/supabase/gotrue/pull/1440
